src/test/resources/feature/2-mittente/1_invioNotifiche/016_133_invioNotificaADomicilioSpeciale.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Mittente invia una notifica digitale con domicilio speciale impostato al destinatario
+
+  @TestSuite
+  @TA_invioNotificaConDomicilioSpecialeImpostato
+  @mittente
+  @invioNotifiche
+
+  Scenario: PN-9247 - Mittente invia una notifica digitale con domicilio speciale impostato al destinatario
+    Given PA - Si effettua la login tramite token exchange, e viene visualizzata la dashboard
+    And Nella pagina Piattaforma Notifiche cliccare sul bottone Invia una nuova notifica
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Informazioni preliminari
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica
+      | oggettoNotifica   | Pagamento rata IMU |
+      | descrizione       | PAGAMENTO RATA IMU |
+      | gruppo            | test-TA-FE-TEST    |
+      | codiceTassonomico | 123456A            |
+      | modalitaInvio     | A/R                |
+    And Cliccare su continua
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Destinatario
+    Then Nella section Destinatario si inseriscono i dati del destinatario
+      | soggettoGiuridico       | PF               |
+      | nomeCognomeDestinatario | Gaio Giulio      |
+      | codiceFiscale           | CSRGGL44L13H501E |
+    And Nella section Destinitario si clicca su "Aggiungi un domicilio digitale" e si inseriscono i dati
+      | digitalAddress | prova@test.it |
+    And Nella section Destinitario si clicca su "Aggiungi un indirizzo fisico" e si inseriscono i dati
+      | indirizzo | Via Roma |
+      | civico    | 20       |
+      | localita  | Milano   |
+      | comune    | Milano   |
+      | provincia | MI       |
+      | cap       | 20147    |
+      | stato     | Italia   |
+    And Cliccare su continua
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Allegati
+    Then Nella section Allegati si carica un atto
+    And Nella section Allegati cliccare sul bottone Invia
+    Then Si visualizza correttamente la frase La notifica è stata correttamente creata
+    And Cliccare sul bottone vai alle notifiche
+    And Si visualizza correttamente la pagina Piattaforma Notifiche
+    And Si verifica che la notifica è stata creata correttamente
+    And Aspetta 180 secondi
+    And Cliccare sulla notifica restituita
+    And Si clicca sul opzione Vedi Dettaglio
+    And Si verifica l'invio della notifica al domicilio speciale inserito "prova@test.it"
+    Then Si verifica che la notifica abbia lo stato "Consegnata"
```

Comment:
group the steps in common with the other files and insert them into the background test file in order to reduce the length of the files